static/scripts/audit-report/audit.ts

```diff
@@ -655,7 +553,6 @@ async function rpcFetcher() {
 async function handleRPCData(data: ChainScanResult) {
   if (data) {
     const { hash, chain } = data as { hash: string; chain: string };
-
     const providerUrl = await getOptimalRPC(chain as Chain);
```

Comment:
Make sure to copy my implementation at https://github.com/ubiquity/pay.ubq.fi/pull/167